src/index.ts

```diff
@@ -35,10 +49,26 @@ export function runInContext(
   context: Context,
   options: Partial<IOptions> = {}
 ): Promise<Result> {
+  function getFirstLine(theProgram: Program) {
+    if (theProgram.body[0] && theProgram.body[0].type === 'ExpressionStatement') {
+      const firstLineOfProgram = theProgram.body[0] as ExpressionStatement
+      const theDirective = (firstLineOfProgram as Directive).directive
+      if (theDirective) {
```

Comment:
I would check for undefined explicitly. 
![maim](https://user-images.githubusercontent.com/829070/53333013-acfcb700-392f-11e9-831b-a87d60dcebff.png)

Reply:
Sorry, could you rephrase? You'd check for undefined on which name?